src/app/helptext/system/bootenv.ts

```diff
@@ -28,7 +28,11 @@ export const helptext_system_bootenv = {
   list_dialog_scrub_action: T("Start Scrub"),
 
   rename_name_placeholder: T("Name"),
-  rename_name_tooltip: T("Rename the existing boot environment."),
+  rename_name_tooltip: T(
+    "Rename the existing boot environment.\
+  Names may include alphanumeric characters as well as\
```

Comment:
"May" implies permission. 
```suggestion
  Allowed characters are alphanumeric,\
```